.github/workflows/static.yaml

```diff
@@ -16,4 +16,3 @@ jobs:
               uses: golangci/golangci-lint-action@v3
               with:
                   version: v1.54
-                  args: -E=gofmt,unused,ineffassign,revive,misspell,exportloopref,asciicheck,bodyclose,depguard,dogsled,durationcheck,errname,forbidigo -D=staticcheck --timeout=30m0s
```

Comment:
why remove all these args?

Reply:
the config has beem moved to .golangci.yml file.